rule-packs/aws-config.json

```diff
@@ -135,13 +135,25 @@
       }
     ]
   },
+  {
+    "name": "rds-instance-public-access-check",
+    "description": "Check whether the Amazon RDS clusters and instances are publicly accessible.",
+    "queries": [
+      {
+        "name": "query0",
+        "query": "Find (aws_db_instance|aws_rds_cluster) with public=true",
+        "version": "v1"
+      }
+    ],
+    "alertLevel": "CRITICAL"
+  },
   {
     "name": "rds-snapshots-public-prohibited",
     "description": "RDS cluster or instance snapshots is public.",
     "queries": [
       {
         "name": "query0",
-        "query": "Find (aws_db_instance|aws_rds_cluster) with public=true",
+        "query": "Find (aws_db_snapshot|aws_db_cluster_snapshot) with public=true",
         "version": "v1"
```

Comment:
What does the `version` signify, if it's not updated when the query changes?

Reply:
that's the j1ql version